src/cow-react/modules/swap/pure/SwapButtons/index.cosmos.tsx

```diff
@@ -0,0 +1,45 @@
+import { WETH_GOERLI } from 'utils/goerli/constants'
+import { CurrencyAmount } from '@uniswap/sdk-core'
+import { SwapButtonState } from '@cow/modules/swap/helpers/getSwapButtonState'
+import { useSelect } from 'react-cosmos/fixture'
+import { SwapButtons, SwapButtonsContext } from './index'
+
+const currency = WETH_GOERLI
+const amount = 200000000
+
+const swapButtonsContext: SwapButtonsContext = {
+  swapButtonState: SwapButtonState.RegularSwap,
+  chainId: 1,
+  wrappedToken: WETH_GOERLI,
+  handleSwap: () => void 0,
+  inputAmount: CurrencyAmount.fromRawAmount(currency, amount * 10 ** 18),
+  wrapUnwrapAmount: CurrencyAmount.fromRawAmount(WETH_GOERLI, 10000000),
+  wrapInputError: undefined,
+  onWrapOrUnwrap: null,
+  onEthFlow: () => void 0,
+  openSwapConfirm: () => void 0,
+  toggleWalletModal: () => void 0,
+  hasEnoughWrappedBalanceForSwap: true,
+}
+
+function useCustomProps(): SwapButtonsContext {
+  const [swapButtonState] = useSelect('swapButtonState', {
+    options: Object.values(SwapButtonState),
+    defaultValue: SwapButtonState.NeedApprove,
+  })
+
+  return {
+    ...swapButtonsContext,
```

Comment:
I don't know if this is something missing in the cosmos fixture props or on the button style itself.

The button styles are quite different:

![Screen Shot 2022-10-27 at 16 59 42](https://user-images.githubusercontent.com/43217/198340805-3d0c20c9-df26-48ce-aad0-ad52c0fde308.png)
![Screen Shot 2022-10-27 at 16 59 34](https://user-images.githubusercontent.com/43217/198340810-f9dddb6a-a797-4489-a89a-1b836102c88b.png)

See it [here](https://cowswap-prs-git-limit-orders-allowance-check-cowswap.vercel.app/cosmos/?fixtureId=%7B%22path%22%3A%22src%2Fcow-react%2Fmodules%2Fswap%2Fpure%2FSwapButtons%2Findex.cosmos.tsx%22%2C%22name%22%3A%22Default%22%7D)

Reply:
Ahhh I see.
I think we should unify all the styles.
Not for this PR though